core/src/androidMain/kotlin/ScannerBuilder.kt

```diff
@@ -1,12 +1,11 @@
 package com.juul.kable
 
 import android.bluetooth.le.ScanSettings
-import com.benasher44.uuid.Uuid
 import com.juul.kable.logs.Logging
 import com.juul.kable.logs.LoggingBuilder
 
 public actual class ScannerBuilder {
-    public actual var services: List<Uuid>? = null
+    public actual var filters: List<Filter>? = null
```

Comment:
We can give people a little bit of time to migrate to the new `filters` property with a deprecation cycle.

```suggestion
    @Deprecated(message = "Replaced by filters property")
    public var services: List<Uuid>?
        set(value) {
            filters = value?.map { Filter.Service(it) }
        }
        get() = filters?.filterIsInstance<Filter.Service>()?.map { it.uuid }

    public actual var filters: List<Filter>? = null
```